### PR TITLE
Fix 394 adguard-vpn.com

### PR DESCRIPTION
**Pull request by Alex-302**
https://github.com/AdguardTeam/HostlistsRegistry/issues/394

### CODE REVIEW

**Review by zloyden — APPROVED** [review]

File: filters/security/filter_52_HageziEncryptedDNSBypass/configuration.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
       ]
     }
   ],
+  "exclusions": ["||adguard.io^", "||dns.adguard.com^", "||dns-family.adguard.com^", "||dns-unfiltered.adguard.com^", "||adguard-dns.com^", "||adguard-vpn.com^"],
```
Comment (slavaleleka) [inline filters/security/filter_52_HageziEncryptedDNSBypass/configuration.json:16]:
```suggestion
  "exclusions": [
    "||adguard.io^",
    "||dns.adguard.com^",
    "||dns-family.adguard.com^",
    "||dns-unfiltered.adguard.com^",
    "||adguard-dns.com^",
    "||adguard-vpn.com^"
  ],
  ```

**Review by AdamWr — APPROVED** [review]

**Comment by hagezi** [comment]
@Alex-302 https://github.com/AdguardTeam/HostlistsRegistry/pull/385#issuecomment-1779882973 ;)

**Comment by Alex-302** [comment]
@hagezi I did not get it:(

**Comment by ameshkov** [comment]
@hagezi yeah, now it's getting caught during generation :)

**Pull request merged**